src/server/auth/arboristClient.js

```diff
@@ -12,11 +12,16 @@ class ArboristClient {
     // Make request to arborist for list of resources with access
     const resourcesEndpoint = `${this.baseEndpoint}/auth/mapping`;
     log.debug('[ArboristClient] listAuthorizedResources jwt: ', jwt);
+    
+    // The POST endpoint checks for the presence of a token and validates it if present. 
+    // For retrocompatibility if the token is not present will use the clientID or username passed in the body. 
+    // We want to avoid using the body values since that could expose 
+    // other clients or usernames that are not the logged in user ones.
```

Comment:
let's remove this comment, since this is all documented in the arborist API docs, and the explanation doesn't really belong in guppy